lexicons/app/bsky/embed/post.json

```diff
@@ -0,0 +1,38 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.embed.post",
+  "description": "An representation of a post embedded in another form of content",
+  "defs": {
+    "main": {
+      "type": "object",
+      "required": ["post"],
+      "properties": {
+        "post": {"type": "ref", "ref": "com.atproto.repo.strongRef"}
+      }
+    },
+    "presented": {
+      "type": "object",
+      "required":  ["post"],
+      "properties": {
+        "post": {"type": "union", "refs": ["#presentedPost", "#presentedNotFound"]}
+      }
+    },
+    "presentedPost": {
+      "type": "object",
+      "required":  ["uri", "cid", "author", "record"],
```

Comment:
Should we have a list (or map) of "relevant blobs" here?

CID -> url map basically (and you can find the CID for the blob from the record)

Reply:
🤔 hmmm not a bad idea!  Paul said he didn't need embeds here yet, but I wouldn't mind adding them if they'll be put to good use.

Approach-wise I think it depends a little bit on whether we go the generic record route or keep it post-specific.  If we keep it post-specific, I would probably tend to include the embeds the same way we do in post views (this object already makes-up a partial post view, so it would involve adding more fields to it).  If we make it generic, I like that proposal a lot.  Actually that is already giving me ideas for the `com.atproto.admin.*` methods in which records appear...

Reply:
yup that makes sense 👍

Reply:
Yeah I'm 99% sure we don't want embeds within embeds. If we change our mind, I think it'll be easy to add as an optional field.